presto-native-execution/presto_cpp/main/tests/PrometheusStatsReporterTest.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "presto_cpp/main/common/PrometheusStatsReporter.h"
+#include <gtest/gtest.h>
+#include <cstring>
+
+namespace ahana {
+
+class AhanaStatsReporterTest : public testing::Test {
```

Comment:
Remove the use of the ahana namespace and use PrometheusStatsReporterTest as the name of the test instead.